fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -0,0 +1,2198 @@
+                !this.eng2AntiIce.get() ? 2 : null,
+            ],
+            codesToReturn: ['308128001', '308128002', '308128003'],
+            memoInhibit: () => false,
+            failure: 2,
+            sysPage: -1,
+            side: 'LEFT',
+        },
+        2900126: { // *HYD  - Blue reservoir overheat
+            flightPhaseInhib: [3, 4, 5, 7, 8],
+            simVarIsActive: this.blueRvrOvht,
+            whichCodeToReturn: () => [
+                0,
+                this.blueElecPumpPBAuto.get() ? 1 : null,
+            ],
+            codesToReturn: ['290012601', '290012602'],
+            memoInhibit: () => false,
+            failure: 2,
+            sysPage: 4,
+            side: 'LEFT',
+        },
+        2900127: { // *HYD  - Yellow reservoir overheat
+            flightPhaseInhib: [3, 4, 5, 7, 8],
+            simVarIsActive: this.yellowRvrOvht,
+            whichCodeToReturn: () => [
+                0,
+                this.ptuAuto.get() ? 1 : null,
+                this.eng2pumpPBisAuto.get() ? 2 : null,
+                !this.yepumpPBisAuto.get() ? 3 : null,
+            ],
+            codesToReturn: ['290012701', '290012702', '290012703', '290012704'],
+            memoInhibit: () => false,
+            failure: 2,
+            sysPage: 4,
+            side: 'LEFT',
+        },
+        2900128: { // *HYD  - Green reservoir overheat
+            flightPhaseInhib: [3, 4, 5, 7, 8],
+            simVarIsActive: this.greenRvrOvht,
+            whichCodeToReturn: () => [
+                0,
+                this.ptuAuto.get() ? 1 : null,
+                this.eng1pumpPBisAuto.get() ? 2 : null,
+            ],
+            codesToReturn: ['290012801', '290012802', '290012803'],
+            memoInhibit: () => false,
+            failure: 2,
+            sysPage: 4,
+            side: 'LEFT',
+        },
+        2900310: { // *HYD  - Blue
+            flightPhaseInhib: [1, 4, 5, 10],
+            simVarIsActive: MappedSubject.create(
+                ([blueRvrLow, blueElecPumpPBAuto, dcESSBusPowered, ac1BusPowered, blueLP, emergencyGeneratorOn]) => !(blueRvrLow || !blueElecPumpPBAuto)
+                    && (!dcESSBusPowered || !ac1BusPowered) && blueLP && !emergencyGeneratorOn,
+                this.blueRvrLow, this.blueElecPumpPBAuto, this.dcESSBusPowered, this.ac1BusPowered, this.blueLP, this.emergencyGeneratorOn,
+            ),
+            whichCodeToReturn: () => [0],
+            codesToReturn: ['290031001'],
+            memoInhibit: () => false,
+            failure: 2,
+            sysPage: 4,
+            side: 'RIGHT',
+        },
+        2900312: { // *HYD  - Green Engine 1 //
+            flightPhaseInhib: [1, 2, 9, 10],
+            simVarIsActive: MappedSubject.create(
+                ([greenLP, greenHydEng1PBAuto, emergencyGeneratorOn]) => greenLP
+                    // && ENG 1 OUT - not implemented
+                    && !greenHydEng1PBAuto && !emergencyGeneratorOn,
```

Comment:
This is just incorrectly inverted.
```suggestion
                    && greenHydEng1PBAuto && !emergencyGeneratorOn,
```